internal/backend/api.go

```diff
@@ -197,11 +207,15 @@ func (h *APIHandler) listAccessRequestHandler(ctx context.Context, input *ListAc
 	if err != nil {
 		return nil, h.loggedError(huma.Error500InternalServerError(fmt.Sprintf("error listing access request for user %s", key.Username), err))
 	}
-
 	return &ListAccessRequestResponse{Body: toListAccessRequestResponseBody(accessRequests)}, nil
 }
 
 func (h *APIHandler) createAccessRequestHandler(ctx context.Context, input *CreateAccessRequestInput) (*CreateAccessRequestResponse, error) {
+	start := time.Now()
+	defer func() {
+		duration := time.Since(start).Seconds()
+		metrics.RecordAPIRequest(createAccessRequestOperation().Method, createAccessRequestOperation().Path, duration)
+	}()
```

Comment:
Please check how to configure middleware with Huma so we avoid duplicating this code in all handlers.